src/foam/nanos/auth/User.js

```diff
@@ -173,7 +173,7 @@ foam.CLASS({
       documentation: 'User\' profile picture.',
       view: {
         class: 'foam.nanos.auth.ProfilePictureView',
-        placeholderImage: 'images/ic-placeholder.png'
+        placeholderImage: 'images/merchant/ic-placeholder.png'
```

Comment:
The merchant folder is part of our nanopay repo, not part of foam. However, an even bigger issue is that we have a direct reference to nanopay for the images service in foam at the moment: https://github.com/foam-framework/foam2/blob/master/src/services#L16

Reply:
Good catch, this property/view can be a refinement on the nanopay side.

Reply:
I'll fix it. Thanks for pointing out.

Reply:
Moved this change to nanopay refinementm and copied the picture from images/merchant to images
https://github.com/nanoPayinc/NANOPAY/pull/6185